src/components/SequenceMenu.vue

```diff
@@ -41,7 +41,7 @@
         SequenceExportModule,
         SequenceExportKind,
     } from '../sequences/SequenceInterface'
-    import {SequenceClassDefault} from '../sequences/SequenceClassDefault'
+    import {SequenceDefault} from '../sequences/SequenceDefault.js'
```

Comment:
Nice, my IDE added a file extension. I will fix.

Reply:
Should be fixed in https://github.com/numberscope/frontscope/pull/242/commits/d8ea8c543c138d66fb9578339adbff6000d96d26.